Orange/widgets/visualize/utils/owlegend.py

```diff
@@ -232,9 +233,10 @@ def __init__(self, color, parent):
         self.__square.setBrush(QBrush(color))
         self.__square.setPen(QPen(QColor(0, 0, 0, 0)))
         self.__square.setParentItem(self)
```

Comment:
pyqt tends to disagree.

RuntimeError: super-class __init__() of type LegendItemSquare was never called